recipes/quantities/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set version = "0.11.1" %}
+
+package:
+    name: quantities
+    version: {{ version }}
+
+source:
+    fn: quantities-{{ version }}.zip
+    url: https://pypi.python.org/packages/source/q/quantities/quantities-{{ version }}.zip
+    md5: f4c6287bfd2e93322b25a7c1311a0243
+
+build:
+    number: 0
+    script: python setup.py install
+
+requirements:
+    build:
+      - python
+      - numpy
+    run:
+      - python
+      - numpy
+
+test:
+    imports:
+      - quantities
+      - quantities.constants
+      - quantities.tests
+      - quantities.units
+
+about:
+    home: http://pythonhosted.org//quantities/
+    license: Python-2.0
```

Comment:
Says BSD on PyPI.

Appears to be [BSD 3-Clause]( http://python-quantities.readthedocs.org/en/latest/user/license.html#license-agreement-for-quantities ).